### ISSUE TITLE
Ubuntu Desktop doesn't start up, complaining that QtWebEngineProcess is not found

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#2378]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.383 - 1.2.405
    OS Version      : Ubuntu 16.04
    R Version       : 3.4.3

### Steps to reproduce the problem

1. Install RStudio Desktop on Ubuntu using build 1.2.383 or later (currently, 1.2.405).
2. Run `rstudio` as a regular user, not root.

### Describe the problem in detail

Starting with build 1.2.383, Ubuntu Desktop does not run as a regular user, with a detailed error starting with:
```
Could not find QtWebEngineProcess
```

It runs successfully as root, though.

### Describe the behavior you expected

RStudio desktop should run as a regular user. 

### Error Information

Below is the error that RStudio generates:
```
vmware@ubuntu:~$ rstudio
Could not find QtWebEngineProcess
[32144:32144:0312/111517.646985:FATAL:zygote_host_impl_linux.cc(182)] Check failed: ReceiveFixedMessage(fds[0], kZygoteBootMessage, sizeof(kZygoteBootMessage), &boot_pid). 
#0 0x7f1026eda0a5 <unknown>
#1 0x7f1026eefeca <unknown>
#2 0x7f1026532fc3 <unknown>
#3 0x7f1026531c3b <unknown>
#4 0x7f1026532219 <unknown>
#5 0x7f102617593a <unknown>
#6 0x7f102617ae76 <unknown>
#7 0x7f1025c98600 <unknown>
#8 0x7f1025c99b25 <unknown>
#9 0x7f1025c2de85 QtWebEngineCore::BrowserContextAdapter::BrowserContextAdapter()
#10 0x7f102c12a4aa QWebEngineProfile::QWebEngineProfile()
#11 0x55803bd32454 <unknown>
#12 0x55803bd2eb14 <unknown>
#13 0x55803bd320fa <unknown>
#14 0x55803bd0bbfd <unknown>
#15 0x55803bd14e48 <unknown>
#16 0x55803bd1b9a0 <unknown>
#17 0x55803bd366cc <unknown>
#18 0x55803bd3732b <unknown>
#19 0x7f102dcae181 QObject::event()
#20 0x7f102d2b563c QApplicationPrivate::notify_helper()
#21 0x7f102d2bc987 QApplication::notify()
#22 0x7f102dc82028 QCoreApplication::notifyInternal2()
#23 0x7f102dc84abb QCoreApplicationPrivate::sendPostedEvents()
#24 0x7f102dcd6b63 <unknown>
#25 0x7f10214dd197 g_main_context_dispatch
#26 0x7f10214dd3f0 <unknown>
#27 0x7f10214dd49c g_main_context_iteration
#28 0x7f102dcd61af QEventDispatcherGlib::processEvents()
#29 0x7f1017b0cb81 <unknown>
#30 0x7f102dc8089a QEventLoop::exec()
#31 0x7f102dc89424 QCoreApplication::exec()
#32 0x55803bd0897c <unknown>
#33 0x7f1024097830 __libc_start_main
#34 0x55803bd0b4ab <unknown>

Received signal 6
#0 0x7f1026eda0a5 <unknown>
#1 0x7f1025c1b141 <unknown>
#2 0x7f1026eda43d <unknown>
#3 0x7f1025303390 <unknown>
#4 0x7f10240ac428 gsignal
#5 0x7f10240ae02a abort
#6 0x7f1026ed9dc5 <unknown>
#7 0x7f1026ef01ed <unknown>
#8 0x7f1026532fc3 <unknown>
#9 0x7f1026531c3b <unknown>
#10 0x7f1026532219 <unknown>
#11 0x7f102617593a <unknown>
#12 0x7f102617ae76 <unknown>
#13 0x7f1025c98600 <unknown>
#14 0x7f1025c99b25 <unknown>
#15 0x7f1025c2de85 QtWebEngineCore::BrowserContextAdapter::BrowserContextAdapter()
#16 0x7f102c12a4aa QWebEngineProfile::QWebEngineProfile()
#17 0x55803bd32454 <unknown>
#18 0x55803bd2eb14 <unknown>
#19 0x55803bd320fa <unknown>
#20 0x55803bd0bbfd <unknown>
#21 0x55803bd14e48 <unknown>
#22 0x55803bd1b9a0 <unknown>
#23 0x55803bd366cc <unknown>
#24 0x55803bd3732b <unknown>
#25 0x7f102dcae181 QObject::event()
#26 0x7f102d2b563c QApplicationPrivate::notify_helper()
#27 0x7f102d2bc987 QApplication::notify()
#28 0x7f102dc82028 QCoreApplication::notifyInternal2()
#29 0x7f102dc84abb QCoreApplicationPrivate::sendPostedEvents()
#30 0x7f102dcd6b63 <unknown>
#31 0x7f10214dd197 g_main_context_dispatch
#32 0x7f10214dd3f0 <unknown>
#33 0x7f10214dd49c g_main_context_iteration
#34 0x7f102dcd61af QEventDispatcherGlib::processEvents()
#35 0x7f1017b0cb81 <unknown>
#36 0x7f102dc8089a QEventLoop::exec()
#37 0x7f102dc89424 QCoreApplication::exec()
#38 0x55803bd0897c <unknown>
#39 0x7f1024097830 __libc_start_main
#40 0x55803bd0b4ab <unknown>
  r8: 00007fff1566b4b0  r9: 00007f102e2e7940 r10: 0000000000000008 r11: 0000000000000206
 r12: 00007fff1566b280 r13: 00000000000000ac r14: 00007fff1566b4b0 r15: 00007fff1566bc20
  di: 0000000000007d90  si: 0000000000007d90  bp: 00007f1024ce1140  bx: 00007fff1566ba00
  dx: 0000000000000006  ax: 0000000000000000  cx: 00007f10240ac428  sp: 00007fff1566b118
  ip: 00007f10240ac428 efl: 0000000000000206 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

**Comment by gtritchie** [comment]
Argh, appears to be a regression in Qt 5.10.1; they've fixed in 5.11 and 5.9, but not 5.10.x.

https://bugreports.qt.io/browse/QTBUG-67023

See:
https://codereview.qt-project.org/#/c/220734/

**Comment by ronblum** [comment]
Confirmed on Ubuntu 16.04 and 17.10.
